geetools/ee_dictionary.py

```diff
@@ -80,3 +82,42 @@ def getMany(self, list: list | ee.List) -> ee.List:
                 d.geetools.getMany(["foo", "bar"]).getInfo()
         """
         return ee.List(list).map(lambda key: self._obj.get(key))
+
+    def toTable(self, valueType: ee.List | ee.Dictionary | Any = Any) -> ee.FeatureCollection:
+        """Convert a ee.Dictionary to a ee.FeatureCollection with no geometries (table).
+
+        The keys will always be stored in `system:index` column.
+
+        Parameters:
+            valueType: this will define how to process the values. In case of
+            ee.List the values will be stored in columns named `value_<position>`.
+            In case of a ee.Dictionary, column names will be created from the keys.
+            For any other type, it will return a table with one feature with
+            one column per key.
+
+        Returns:
+            a ee.FeatureCollection in which the keys of the ee.Dictionary are
+            in the `system:index` and the values are in new columns.
+        """
+
+        def features_from_dict(key, value) -> ee.Feature:
+            key = ee.String(key)
+            feat = ee.Feature(None, {"system:index": key})
+            return feat.set(ee.Dictionary(value))
```

Comment:
Alternative building blocks. I think setting all the props at once makes it easier to understand what the function is doing. As there is no line gain, it's up to you to decide which one is better

```suggestion
            index = {"system:index": ee.String(key)}
            props = ee.Dictionary(value).combine(index)
            return ee.Feature(None, props)
```